.changesets/breaking_simon_introspection_max_depth.md

```diff
@@ -0,0 +1,15 @@
+### Limit the depth of introspection queries ([PR #6601](https://github.com/apollographql/router/pull/6601))
+
+The [schema-intropsection schema](https://spec.graphql.org/draft/#sec-Schema-Introspection.Schema-Introspection-Schema) is recursive: a client can query the fields of the types of some other fields, and so on arbitrarily deep. This can produce responses that grow much faster than the size of the request.
+
+To protect against abusive Router now refuses to execute introspection queries that nest list fields too deep and returns an error instead. The criteria matches `MaxIntrospectionDepthRule` in graphql-js, but may change in future versions.
```

Comment:
```suggestion
To protect against abuse Router now refuses to execute introspection queries that nest list fields too deep and returns an error instead. The criteria matches `MaxIntrospectionDepthRule` in graphql-js, but may change in future versions.
```

Reply:
I a word, this was meant to be "To protect against abusive requests". Which do you prefer?

Reply:
Perfectly happy with abusive requests :)